rest/src/main/java/fi/otavanopisto/pyramus/rest/OrganizationRESTService.java

```diff
@@ -298,4 +300,32 @@ public Response deleteOrganization(@PathParam("ID") Long id, @DefaultValue ("fal
     return Response.noContent().build();
   }
   
+  @Path("/{ID:[0-9]*}/contactPerson")
```

Comment:
I would use plural contactPerson**s** as it's a list of things instead of a singular thing.